src/native-automation/client/key-press/utils.ts

```diff
@@ -7,6 +8,10 @@ import { changeLetterCase, getActualKeysAndEventKeyProperties } from '../../../c
 import { getModifiersState } from '../utils';
 import getKeyCode from '../../../client/automation/utils/get-key-code';
 
+const SHORTCUT_TO_COMMANDS_MAP = {
```

Comment:
I propose implementing the `ctrl+c`, `ctrl+x` and `ctrl+v` commands at once.

Reply:
Since current testcafe version doesn't support the `ctrl+c`, `ctrl+x` and ctrl+v commands lets implement this functionality as enhancement in one of the next testcafe releases